src/app/feed/feed-news/feed-news.component.html

```diff
@@ -1,3 +1,11 @@
+<div class="message">
+  <!-- Hook up a message if needed like google does "About xxxxx results (y.yy seconds)" -->
```

Comment:
Please make comment simple like `<!-- Message about total results-->` we are doing it for users' experience. No need to mention who does that and who does not.